build.sbt

```diff
@@ -14,12 +14,7 @@ ThisBuild / tlSonatypeUseLegacyHost := false
 val Scala213 = "2.13.8"
 ThisBuild / crossScalaVersions := Seq(Scala213, "2.12.15", "3.0.2")
 ThisBuild / scalaVersion := Scala213 // the default Scala
-
-ThisBuild / homepage := Some(url("https://github.com/TimWSpence/cats-stm"))
-
-ThisBuild / scmInfo := Some(
-  ScmInfo(url("https://github.com/TimWSpence/cats-stm"), "git@github.com:TimWSpence/cats-stm.git")
-)
+ThisBuild / tlJdkRelease := Some(8)
```

Comment:
There you go :)

Reply:
Does this set the javac target option?

Reply:
Yes.

Reply:
Well, actually no :) it sets the `--release` flag which has superseded that, but anyway.